sandbox/src/routes/r4/retrieveBinaryHelper.js

```diff
@@ -0,0 +1,22 @@
+module.exports = [
+  /**
+   * Sandbox implementation for retrieveBinary (R4) endpoint helper, 
+   * with an ObjectStore 'mock', allowing redirection and simple file retrieval.
+   */
+  {
+    method: 'GET',
+    path: '/ObjectStore/RetrieveBinary/{fileDownloadUuid}',
+    handler: (request, h) => {
+
+      const uuid = request.params.fileDownloadUuid
+      const filename = 'example_attachment.pdf'
+      const filePath = `./r4/requestUrlForFileDownload/${filename}`
+      const responseCode = 200
+
+      if (uuid === 'd497bbe3-f88b-45f1-b3d4-9c563e4c0f5f') {
```

Comment:
As highlighted above I wonder if it's worth supporting any ID in the legacy or UUID format here?